docs/DeveloperGuide.adoc

```diff
@@ -77,6 +77,40 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: Rename tags
+
+*MSS*
+
+. User requests to rename a tag
+. AddressBook shows a list of tags
+. User enters the tag to be changed
+. AddressBook prompts user for the new tag name
+. User enters the new tag name
+. Addressbook confirms the tag name with user
```

Comment:
If you have this step, the next one should be something like `User confirms the rename operation`.
Remember, use cases depict the interaction between the user and the system